.travis.yml

```diff
@@ -45,6 +45,10 @@ matrix:
     fast_finish: true
     include:
 
+        # Try with pint
+        - python: 3.6
```

Comment:
We might also add a test for 2.7, but pint has definitely more problems with 3.6 than 2.7, so this test should be the critical one.
Adding a test is a matter of copying and pasting that line :). If you would prefer it to be there, I'll add it.